lib/avo/fields/concerns/reload_icon.rb

```diff
@@ -5,11 +5,11 @@ module ReloadIcon
         extend ActiveSupport::Concern
 
         included do
-          attr_accessor :reload_button
+          attr_accessor :reloadable
         end
 
         def reload_icon_enabled?
```

Comment:
How about this name?

```suggestion
        def reloadable?
```

Reply:
Sure, I will change it.